tsconfig.json

```diff
@@ -44,10 +44,7 @@
     //   "node_modules/@types",
     //   "packages/core/types"
     // ],
-    "types": [                                /* Type declaration files to be included in compilation. */
```

Comment:
Better to just install the `@types` packages and let TS pick them up automatically.